vertical-pod-autoscaler/pkg/recommender-external/input/metrics/external_metrics_client_test_util.go

```diff
@@ -15,3 +15,82 @@ limitations under the License.
 */
 
 package metrics
```

Comment:
can we move this file into a `test` or `fake ` package to clearly isolate what is used for test and what is not. Of course some symbols have `Test` in clear separation in package is better for code exploration and static code analysis later.

Reply:
done